server.py

```diff
@@ -1,296 +0,0 @@
-
-    glitz_comments = get_glitz_comments(glitz_feeds)  
-
-    return render_template("profile.html", glitz_comments=glitz_comments
-                            ,grids=grids)                                                   
-
-
-
-@app.route('/register', methods=['GET'])
-def register_form():
-    """Show form for user signup."""
-
-    return render_template("register_form.html")
-
-
-@app.route('/register', methods=['POST'])
-def register_process():
-    """Process registration."""
-
-    # Get form variables
-    lname = request.form["lname"]
-    fname = request.form["fname"]
-    email = request.form["email"]
-    password = request.form["password"]
-    
-    new_user = User(fname=fname, lname=lname)
-
-    db.session.add(new_user)
-    db.session.commit()
-    
-    new_password = encrypt_password(password)
-
-    new_login = Login(user_id=new_user.user_id, email=email, password=new_password)
-
-    db.session.add(new_login)
-    db.session.commit()
-
-    session["user_id"] = new_login.user_id
-
-    return redirect("/")
-
-# @app.route("/<int:user_id>")
-# def user_detail(user_id):
-#     """Show info about user."""
-
-#     user = User.query.options(db.joinedload('logins')).get(user_id)
-#     return render_template("profile.html", user=user)
-
-
-
-@app.route('/login', methods=['GET'])
-def login_form():
-    """Show login form."""
-
-    return render_template("login_form.html")
-
-
-@app.route('/login', methods=['POST'])
-def login_process():
-    """Process login."""
-
-    # Get form variables
-    email = request.form["email"]
-    password = request.form["password"]
-
-    encrypted_password = encrypt_password(password)
-
-    logged_in_user = Login.query.filter_by(email=email).first()
-
-    if not logged_in_user:
-        flash("No such user")
-        return redirect("/register")
-
-    if logged_in_user.password != encrypted_password:
-        flash("Incorrect password")
-        return redirect("/login")
-
-    session["user_id"] = logged_in_user.user_id
-
-    return redirect("/")
-    # return redirect("/%s" % logged_in_user.user_id)
-
-
-@app.route('/logout')
-def logout():
-    """Log out."""
-
-    del session["user_id"]
-    return redirect("/")
-
-@app.route('/upload_glitz', methods=['POST'])
-def upload_glitz():
-    """Upload Glitz."""
-
-    if not is_user_logged_in(session):
-            return redirect("/login")
-
-    # Get image 
-    if request.method == 'POST':
-        # check if the post request has the file part
-        
-        if 'glitzToUpload' not in request.files:
-            # flash('No file part')
-            return redirect("/")
-
-        file = request.files['glitzToUpload']
-
-        # if user does not select file, browser also
-        # submit a empty part without filename
-        if file.filename == '':
-            # flash('No selected file')
-            return redirect("/")
-
-        if file and allowed_file(file.filename):
-            filename = secure_filename(file.filename)
```

Comment:
If you don't need to preserve the filename, I'd suggest not doing it.  You might want to just save it as the file_id.extension or something.  I suggest this mainly because some of the images you checked in are too long.  I got this trying to check out the repo:

> stridera@fluttershy:~/code$ git clone git@github.com:stridera/Project-Glitzy.git
Cloning into 'Project-Glitzy'...
remote: Counting objects: 120, done.
remote: Compressing objects: 100% (84/84), done.
remote: Total 120 (delta 52), reused 102 (delta 34), pack-reused 0
Receiving objects: 100% (120/120), 20.65 MiB | 6.31 MiB/s, done.
Resolving deltas: 100% (52/52), done.
**error: unable to create file glitz/Beautiful-Model-Gisele-Bundchen-Modeling-For-Salvatore-Ferragamo-Spring-Summer-Fashion-Advertising-Campaign-As-The-Highest-Paid-Model-In-The-World.jpg: File name too long
fatal: unable to checkout working tree**
warning: Clone succeeded, but checkout failed.
You can inspect what was checked out with 'git status'
and retry the checkout with 'git checkout -f HEAD'